### PR TITLE
Add Code tab to Wizard page

### LINKED ISSUE #1080: Add Code tab to Wizard page

### PR DESCRIPTION
**Pull request by marcialfps**
Closes #1080

### CODE REVIEW

File: website/screens/components/wizard/code/examples/controlled.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import { DxcWizard, DxcInset } from "@dxc-technology/halstack-react";
+import { useState } from "react";
+
+const code = `() => {
+    const [myCurrentStep, setMyCurrentStep] = useState(2);
+    const onStepClick = i => {
+        setMyCurrentStep(i);
+    };
+
+    const userIcon = (
```
Comment (Lisa159) [inline website/screens/components/wizard/code/examples/controlled.ts:10]: what do you think about moving this icon to an external file to not have so much code in the example?

File: website/screens/components/wizard/usage/examples/content.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import { DxcWizard, DxcInset } from "@dxc-technology/halstack-react";
+
+const code = `() => {
+    const userIcon = (
```
Comment (Lisa159) [inline website/screens/components/wizard/usage/examples/content.ts:4]: what do you think about moving this icon to an external file to have a cleaner code?

File: website/screens/components/wizard/usage/examples/description.ts [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import { DxcWizard, DxcInset } from "@dxc-technology/halstack-react";
+
+const code = `() => {
+    const userIcon = (
```
Comment (Lisa159) [inline website/screens/components/wizard/usage/examples/description.ts:4]: what do you think about moving this icon to an external file to have a cleaner code?

File: website/screens/components/wizard/code/examples/basicUsage.ts [hunk 4]
```diff
@@ -0,0 +1,32 @@
+import { DxcWizard, DxcInset } from "@dxc-technology/halstack-react";
+
+const code = `() => {
+    return (
+      <DxcInset space="medium">
```
Comment (Lisa159) [inline website/screens/components/wizard/code/examples/basicUsage.ts:5]: In the rest of the examples, we are using space="large".

File: website/screens/components/wizard/usage/examples/validation.ts [hunk 5]
```diff
@@ -0,0 +1,42 @@
+import { DxcWizard, DxcInset, DxcStack } from "@dxc-technology/halstack-react";
+
+const code = `() => {
+    return (
+      <DxcInset space="medium">
+        <DxcStack gutter="medium">
```
Comment (Lisa159) [inline website/screens/components/wizard/usage/examples/validation.ts:6]: What do you think about gutter="large"?

File: website/screens/components/wizard/code/examples/basicUsage.ts (hunk 4 above)
Comment (marcialfps) [reply website/screens/components/wizard/code/examples/basicUsage.ts]: Done!

File: website/screens/components/wizard/usage/examples/validation.ts (hunk 5 above)
Comment (marcialfps) [reply website/screens/components/wizard/usage/examples/validation.ts]: Done! ✌🏼

**Review by Lisa159 — APPROVED** [review]

**Pull request merged**